frontend/app/handlers.ts

```diff
@@ -1,19 +1,47 @@
+import {
+    crypto_pwhash_str,
+    crypto_pwhash_str_verify,
+    crypto_pwhash_OPSLIMIT_INTERACTIVE,
+    crypto_pwhash_MEMLIMIT_INTERACTIVE,
+} from "libsodium-wrappers-sumo";
+
 // Make API call to server to check if password was found in breached dataset
 export const checkSecurity = async (password: string) => {
     try {
-        const response = await fetch("http://localhost:18080/intersection", {
-            method: "POST",
-            mode: "cors",
-            headers: {
-                "Access-Control-Allow-Headers": "*", // cors setting
-                "Content-Type": "application/json"
-            },
-            body: password
-        })
+        // hash the password
+        const passwordHash = crypto_pwhash_str(
```

Comment:
**question**: In the example of (libsodium documentation)[https://libsodium.gitbook.io/doc/password_hashing/default_phf], they provide a `char[]` and `strlen(password)` to store the password (see Example 2: Password Storage). Does initializing a constant get rid of the need to have a `char[]` since it will return the result to passwordHash? Does the OPSLIMIT_INTERACTIVE eliminate the need for `strlen(password)` because it dynamically checks the password size?